resources/customer.js

```diff
@@ -46,4 +46,17 @@ Customer.prototype.accountActivationUrl = function accountActivationUrl(id) {
   }).then(body => body.account_activation_url);
 };
 
+/**
+ * Generates a invite email for a customer.
+ *
+ * @param {Number} id Customer ID
+ * @param {Object} params Optional params for adjusting the sent email
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+Customer.prototype.sendInvite = function sendInvite(id, params) {
+  const url = this.buildUrl(`${id}/send_invite`);
+  return this.shopify.request(url, 'POST', 'customer_invite', params).then(body => body.customer_invite);
```

Comment:
This should be

```js
return this.shopify.request(url, 'POST', 'customer_invite', params || {});
```

or it won't work with default invite.
The `then()` part is wrong as the promise is already resolved with ` body[key]` if the `key` argument is used in `Shopify.prototype.request()`.